tests/models/plants/conftest.py

```diff
@@ -26,6 +26,13 @@ def plants_config():
         out_initial_file_name = "model_at_start.nc"
         out_final_file_name = "model_at_end.nc"
 
+        [core.layers]
+        canopy_layers = 10
+        soil_layers = [-0.5, -1.0]
```

Comment:
Recently switched the default topsoil depth to 25cm, so might be good to change this to `[-0.25, -1.0]`.

More broadly we should think about whether this topsoil layer should defined based on the global constant (see #348), or alternatively whether that constant should be removed in favour of just using these layer heights. But that's out of the scope of this PR.